doc/user-guide/alt-deploy.md

```diff
@@ -0,0 +1,66 @@
+## Deploying via olm
+
+- Clone the repo
+```
+git clone git@github.com:red-hat-storage/lvm-operator.git && cd lvm-operator
+```
+
+### Deploying pre-built images
+
+- Quick install using pre-built container images via OLM.
```

Comment:
There are no pre-built images at present.

Reply:
ack, remove this section or temporarily push images to somewhere accessible?